css/danti.css

```diff
@@ -0,0 +1,306 @@
+.contener{
+    display: grid;
+    height: 100vh;
+    grid-template-columns:0.5fr 1fr 1fr 1fr 1fr 0.5fr;
+    grid-template-rows: 0.5fr 3fr 3fr 0.5fr ;
+    grid-template-areas: 
+        "nav nav nav nav nav nav"
+        ". form form img img img"
+        ". form form img img img"
+        ". hr hr hr hr hr"
+        ". foot foot foot foot foot"
```

Comment:
Because you're just stacking the various sections of the page on top of each other, you don't really need them to be part of the same grid layout.

The only block here that needs more than one column is the row with the form on the left and the image on the right, so you could just have that block be a grid. This would make this code more concise and easier to understand :nerd_face: